communication/src/dtls_message_channel.cpp

```diff
@@ -405,7 +407,6 @@ ProtocolError DTLSMessageChannel::establish(uint32_t& flags, uint32_t app_state_
 ProtocolError DTLSMessageChannel::notify_established()
 {
 	sessionPersist.make_persistent();
-	sessionPersist.save(callbacks.save);
```

Comment:
So we don't actually need to store the `persistent` flag itself in the session's persistent storage, right?

Reply:
This was an erroneous delete - will fix up, thanks for spotting it!